packages/nx-plugin/generators.json

```diff
@@ -19,7 +19,7 @@
     "app": {
       "factory": "./src/generators/app/compat",
       "schema": "./src/generators/app/schema.json",
-      "description": "Add Angular Three with proper packages and config"
+      "description": "Generates an AnalogJs application"
```

Comment:
```suggestion
      "description": "Generates an Analog application"
```

Reply:
No problem, this was copied from the Nx generator description above, so I will change both.